packages/html-reporter/src/metadataView.tsx

```diff
@@ -73,10 +86,10 @@ export const MetadataView: React.FC<Metadata> = metadata => {
           icon='externalLink'
         />
       }
-      {metadata['generatedAt'] &&
+      {metadata.generatedAt &&
```

Comment:
Let's keep them uniform, always access as string keys, all lower-case, '.' for namespaces. Core can be no namespace, 'timestamp' in this case.